src/pagination/pagination.ts

```diff
@@ -11,39 +11,43 @@ import {NgbPaginationConfig} from './pagination-config';
   host: {'role': 'navigation'},
   template: `
     <ul [class]="'pagination' + (size ? ' pagination-' + size : '')">
-      <li *ngIf="boundaryLinks" class="page-item"
-        [class.disabled]="!hasPrevious() || disabled">
-        <a aria-label="First" i18n-aria-label="@@ngb.pagination.first-aria" class="page-link" href
-          (click)="!!selectPage(1)" [attr.tabindex]="(hasPrevious() ? null : '-1')">
+      <li *ngIf="boundaryLinks" title="{{!hasPrevious() || disabled ? 'First Page (disabled)' : 'First Page'}}" class="page-item"
+          [class.disabled]="!hasPrevious() || disabled">
+        <a aria-label="First Page" id="paginationPageFirst" i18n-aria-label="@@ngb.pagination.first-aria" class="page-link" href
```

Comment:
hardcoded ids won't be unique for the document, if you have multiple pagination components displayed

Reply:
Yep, we definitively do NOT want to bring `id`s to our widgets - please remove.